src/sidebar/host-config.js

```diff
@@ -43,6 +43,9 @@ export default function hostPageConfig(window) {
     // Fetch config from a parent frame.
     'requestConfigFromFrame',
 
+    // Number of ancestor frames that the parent frame resides above this sidebar
+    'configFrameAncestorLevel',
```

Comment:
I think this change is left over from the first iteration of this PR and no longer applicable?